### PR TITLE
Web Modeler 0.5.0 beta

### LINKED ISSUE #85: Platform won't start with Web Modeler due to Identity container being unhealthy
To reproduce:
* clone repo
* Run `docker-compose -f docker-compose.yaml -f docker-compose-web-modeler-beta.yaml pull` to pull latest snapshots
* Run `docker-compose -f docker-compose.yaml -f docker-compose-web-modeler-beta.yaml up -d`

Start fails because Identity container is unhealthy.

Log looks ok:
```
 :: Spring Boot ::                (v3.0.1)
2023-02-13 11:55:53.514  INFO 1 --- [           main] i.c.i.Application                        : Starting Application using Java 17.0.5 with PID 1 (/app/identity.jar started by root in /app)
2023-02-13 11:55:53.528  INFO 1 --- [           main] i.c.i.Application                        : The following 1 profile is active: "keycloak"
2023-02-13 11:55:55.983  WARN 1 --- [           main] ocalVariableTableParameterNameDiscoverer : Using deprecated '-debug' fallback for parameter name resolution. Compile the affected code with '-parameters' instead or avoid its introspection: io.camunda.identity.config.IdentityCommon
2023-02-13 11:55:55.993  WARN 1 --- [           main] ocalVariableTableParameterNameDiscoverer : Using deprecated '-debug' fallback for parameter name resolution. Compile the affected code with '-parameters' instead or avoid its introspection: io.camunda.identity.security.spring.filter.FilterExceptionHandler
2023-02-13 11:55:56.478  WARN 1 --- [           main] ocalVariableTableParameterNameDiscoverer : Using deprecated '-debug' fallback for parameter name resolution. Compile the affected code with '-parameters' instead or avoid its introspection: io.camunda.identity.impl.keycloak.config.record.KeycloakClient
2023-02-13 11:55:56.748  WARN 1 --- [           main] .s.s.UserDetailsServiceAutoConfiguration : 
Using generated security password: bcbbeb31-9b58-4292-8866-ecf85c19272d
This generated password is for development use only. Your security configuration must be updated before running your application in production.
2023-02-13 11:55:58.537  INFO 1 --- [           main] i.c.i.Application                        : Started Application in 5.504 seconds (process running for 6.269)
```

Tested with today's (13.02.2023) snapshot on Linux Ubuntu 22.04.1 LTS

### PR DESCRIPTION
**Pull request by jfriedenstab**
Updates to latest version of Web Modeler.
Closes #85.

### CODE REVIEW

File: docker-compose-web-modeler-beta.yaml [hunk 1]
```diff
@@ -23,8 +23,6 @@ services:
   modeler-db:
     container_name: modeler-db
     image: postgres:14.2-alpine
-    ports:
-      - "5433:5432"
```
Comment (jfriedenstab) [inline docker-compose-web-modeler-beta.yaml:27]: Why I removed the exposed port: https://camunda.slack.com/archives/C6GBS7DKL/p1676474625791639

File: docker-compose.yaml [hunk 2]
```diff
@@ -171,11 +171,11 @@ services:
       KEYCLOAK_CLIENTS_0_PERMISSIONS_0_RESOURCE_SERVER_ID: operate-api
       KEYCLOAK_CLIENTS_0_PERMISSIONS_0_DEFINITION: read:*
     healthcheck:
-      test: [ "CMD", "curl", "-f", "http://localhost:8084/actuator/health" ]
+      test: [ "CMD", "curl", "-f", "http://localhost:8082/actuator/health" ]
```
Comment (jfriedenstab) [inline docker-compose.yaml:174]: Not sure why `8084` was used before 🤷🏻. We also use `8082` here: https://github.com/camunda/web-modeler/blob/16a41d3c94fecb20d8665b59bd16dc9414ebfb0b/docker-compose.sm.yml#L100

Comment (pihme) [reply docker-compose.yaml:174]: Doesn't make a difference on my machine. Container still starts as unhealthy

**Review by pihme — CHANGES_REQUESTED** [review]

File: docker-compose.yaml (hunk 2 above)
Comment (jfriedenstab) [reply docker-compose.yaml:174]: Huh, that's weird! Can you check the output of this command?
```sh
docker exec -it identity curl -f http://localhost:8082/actuator/health
```

Comment (pihme) [reply docker-compose.yaml:174]: Is reported as `{"status":"UP"}

Comment (pihme) [reply docker-compose.yaml:174]: Nevermind, I messed something up when checking out the branch

**Review by pihme — APPROVED** [review]

**Pull request merged**